src/sql/sql-multi/sql-multi.spec.ts

```diff
@@ -2316,4 +2316,184 @@ describe('getSqlRefs', () => {
       }
     `);
   });
+  it('IS function', () => {
+    const sql = `REGEXP_EXTRACT("cityName", 'San') IS NULL`;
+
+    expect(parser(sql).toString()).toMatchInlineSnapshot(
+      `"REGEXP_EXTRACT(\\"cityName\\", 'San') IS NULL"`,
+    );
+
+    expect(parser(sql)).toMatchInlineSnapshot(`
+      SqlMulti {
+        "arguments": Array [
+          SqlFunction {
+            "arguments": Array [
+              SqlRef {
+                "column": "cityName",
+                "innerSpacing": Object {},
+                "namespace": undefined,
+                "namespaceQuotes": undefined,
+                "quotes": "\\"",
+                "table": undefined,
+                "tableQuotes": undefined,
+                "type": "ref",
+              },
+              SqlLiteral {
+                "innerSpacing": Object {},
+                "stringValue": "San",
+                "type": "literal",
+                "value": "San",
+              },
+            ],
+            "decorator": undefined,
+            "filterKeyword": undefined,
+            "functionName": "REGEXP_EXTRACT",
+            "innerSpacing": Object {
+              "postDecorator": "",
+              "postFilterKeyword": "",
+              "postFilterLeftParen": "",
+              "postLeftParen": "",
+              "postName": "",
+              "postWhereKeyword": "",
+              "preFilter": "",
+              "preFilterRightParen": "",
+              "preRightParen": "",
+            },
+            "separators": Array [
+              Separator {
+                "left": "",
+                "right": " ",
+                "separator": ",",
+              },
+            ],
+            "type": "function",
+            "whereExpression": undefined,
+            "whereKeyword": undefined,
+          },
+          "NULL",
+        ],
+        "expressionType": "Comparison",
+        "innerSpacing": Object {},
+        "separators": Array [
+          Separator {
+            "left": " ",
+            "right": " ",
+            "separator": "IS",
+          },
+        ],
+        "type": "multi",
+      }
+    `);
+  });
+
+  it('IS Not function', () => {
+    const sql = `X IS NOT NULL`;
+
+    expect(parser(sql).toString()).toMatchInlineSnapshot(`"X IS NOT NULL"`);
+
+    expect(parser(sql)).toMatchInlineSnapshot(`
+      SqlMulti {
+        "arguments": Array [
+          SqlRef {
+            "column": "X",
+            "innerSpacing": Object {},
+            "namespace": undefined,
+            "namespaceQuotes": undefined,
+            "quotes": "",
+            "table": undefined,
+            "tableQuotes": undefined,
+            "type": "ref",
+          },
+          "NULL",
```

Comment:
"NULL" should not be a freestanding expression in `arguments` they are `SqlBase[]` anyway... how did this even get through? NULL is a literal expression (just like and TRUE, FALSE)... Could you add tests for all of those please